### PR TITLE
feat: Dedicated tab for campaigns 

### LINKED ISSUE #2716: feat: Dedicated Tab for campaigns
At the moment campaigns reside inside inbox settings, Let's move it out to a dedicated tab on the sidebar for easy access

### PR DESCRIPTION
**Pull request by muhsin-k**
Fixes #2716 
Screenshot 2021-08-04 at 3 44 42 PM

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/campaigns/campaigns.routes.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      props: {
+        headerTitle: 'CAMPAIGN.ONE_OFF.HEADER',
+        icon: 'ion-radio-waves',
+      },
+      children: [
+        {
+          path: 'one_off',
+          name: 'one_off',
+          roles: ['administrator'],
+          component: Index,
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/settings/campaigns/campaigns.routes.js:39]:
```suggestion
          component: { ...Index },
```

File: app/javascript/dashboard/routes/dashboard/settings/campaigns/AddCampaign.vue [hunk 2]
```diff
@@ -261,13 +256,26 @@ export default {
...
+      try {
+        const response = await this.$store.dispatch('inboxMembers/get', {
+          inboxId: this.selectedInbox,
+        });
+        const {
+          data: { payload: inboxMembers },
+        } = response;
+        this.senderList = inboxMembers;
+      } catch (error) {
+        //  Handle error
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/settings/campaigns/AddCampaign.vue:269]: Show alert ?

File: app/javascript/dashboard/routes/dashboard/settings/campaigns/EditCampaign.vue [hunk 3]
```diff
@@ -176,22 +185,46 @@ export default {
...
+      try {
+        const response = await this.$store.dispatch('inboxMembers/get', {
+          inboxId: this.selectedInbox,
+        });
+        const {
+          data: { payload: inboxMembers },
+        } = response;
+        this.senderList = inboxMembers;
+      } catch (error) {
+        //  Handle error
```
Comment (nithindavid) [inline app/javascript/dashboard/routes/dashboard/settings/campaigns/EditCampaign.vue:199]: Show alert ?

**Review by nithindavid — COMMENTED** [review]
Have some minor comments

**Comment by muhsin-k** [comment]
@sojan-official Could you please do the QA?

**Review by nithindavid — APPROVED** [review]

**Review by iamsivin — APPROVED** [review]

**Pull request merged**